### PR TITLE
Add confirmButtonText and cancelButtonText config options

### PR DESCRIPTION
**Pull request by joeveiga**
## Summary

Support for customizing the actions text for `<ngx-alert-dialog />` component. Added `confirmButtonText` and `cancelButtonText` inputs to the component. 

## Checklist

- [x] \*Added unit tests
- [x] \*Added a code reviewer
- [ ] Added changes to `/projects/swimlane/ngx-ui/CHANGELOG.md` under HEAD (Unreleased)
- [x] Updated the demo page
- [ ] Included screenshots of visual changes

_\*required_

### CODE REVIEW

File: projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.spec.ts [hunk 1]
```diff
@@ -28,13 +29,28 @@ describe('AlertComponent', () => {
     component = fixture.componentInstance;
     component.longPress = false;
     component.type = AlertTypes.Alert;
-    fixture.detectChanges();
+    // fixture.detectChanges();
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.spec.ts:32]: Comment

File: projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.ts [hunk 2]
```diff
@@ -61,6 +61,8 @@ import { AlertTypes } from './alert-types.enum';
 export class AlertComponent extends DialogComponent implements AfterViewInit {
   @Input() type: AlertTypes;
   @Input() data: any = '';
+  @Input() confirmButtonText: string;
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.ts:64]:
```suggestion
  @Input() confirmButtonText: string = 'Ok';
```

File: projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.ts [hunk 3]
```diff
@@ -61,6 +61,8 @@ import { AlertTypes } from './alert-types.enum';
...
+  @Input() cancelButtonText: string;
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.ts:65]:
```suggestion
  @Input() cancelButtonText: string = 'Cancel';
```

File: projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.html [hunk 4]
```diff
@@ -33,10 +28,10 @@ <h1 *ngIf="title" [innerHTML]="title"></h1>
     <div class="ngx-dialog-footer" *ngIf="type !== AlertTypes.Alert">
       <ng-container *ngIf="!longPress">
         <button type="button" class="btn btn-primary" (click)="onOkClick()">
-          Ok
+          {{ confirmButtonText || 'Ok' }}
         </button>
         <button type="button" class="btn" (click)="onCancelClick()">
-          Cancel
+          {{ cancelButtonText || 'Cancel' }}
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.html:34]:
```suggestion
          {{ cancelButtonText }}
```

File: projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.html [hunk 5]
```diff
@@ -33,10 +28,10 @@ <h1 *ngIf="title" [innerHTML]="title"></h1>
     <div class="ngx-dialog-footer" *ngIf="type !== AlertTypes.Alert">
       <ng-container *ngIf="!longPress">
         <button type="button" class="btn btn-primary" (click)="onOkClick()">
-          Ok
+          {{ confirmButtonText || 'Ok' }}
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.html:31]:
```suggestion
          {{ confirmButtonText }}
```

File: projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.html [hunk 6]
```diff
@@ -14,13 +14,8 @@
...
       </button>
-      <button
-        *ngIf="type === AlertTypes.Alert"
-        type="button"
-        class="btn close-button"
-        (click)="onOkClick()"
-      >
-        Ok
+      <button *ngIf="type === AlertTypes.Alert" type="button" class="btn close-button" (click)="onOkClick()">
+        {{ confirmButtonText || 'Ok' }}
```
Comment (duredhel13) [inline projects/swimlane/ngx-ui/src/lib/components/dialog/alert/alert.component.html:18]:
```suggestion
        {{ confirmButtonText }}
```

**Review by duredhel13 — APPROVED** [review]

**Review by nelsonGuti — APPROVED** [review]
LGTM!

**Pull request merged**